### ISSUE TITLE
The focus indicator is not clearly visible on the checkboxes of the table present under 'Angular Grid Row selection' page.

### ISSUE THREAD

**Issue by msftedad** [angular-ui/ui-grid#7238]
**Test Environment**
OS: Windows_11
Version: 21H2
OS Build: 22000.795
Browser: Version 103.0.1264.71 (Official Build) (64-bit)
URL: https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/row-selection

**Repro Steps:**
1. Open https://www.infragistics.com/products/ignite-ui-angular
2. Angular Row selection row pen page will open.
3. Reach to the main landmark content where ' Examples' tab is selected.
4. With 6-tab press, reach to the column header of the available table.
5. Verify the issue that the focus is clearly visible on the check box controls.

**Actual Behavior:**
The focus indicator is not clearly visible on the checkboxes of the table present under the 'Angular Grid Row Selection ' page.

**Expected Behavior:**
The focus indicator should be clearly visible on the checkboxes of the table present under the 'Angular Grid Row Selection ' page.
<img width="1896" alt="194482 MAS 2 4 7" src="https://user-images.githubusercontent.com/93735775/184089196-11cd5aed-4a40-49ab-a015-bea6e2f05c31.png">
[The focus indicator is not clearly visible . - Copy.zip](https://github.com/angular-ui/ui-grid/files/9307647/The.focus.indicator.is.not.clearly.visible.-.Copy.zip)

**Comment by mportuga** [comment]
This defect is not for UI-Grid, but for Ignite UI, which is managed by a completely different team.
